### ISSUE TITLE
what is the train set?

### ISSUE THREAD

**Issue by haoren55555** [xingchenzhang/VIFB#17]
It seems that only test set is provided, then which train set should I use to train the model?

**Comment by xingchenzhang** [comment]
Yes, VIFB only contains a test set, no training set. 

You can use other RGB-T datasets for training, e.g., RoadScene and MFNet.
